server/chat-commands/info.ts

```diff
@@ -2463,7 +2463,6 @@ export const commands: ChatCommands = {
 	},
 	showimagehelp: [`/showimage [url], [width], [height] - Show an image. Any CSS units may be used for the width or height (default: px). If width and height aren't provided, automatically scale the image to fit in chat. Requires: # & ~`],
 
-	'!pi': true,
```

Comment:
This should be in a separate pull request. `!pi` has nothing to do with room events.